pages/osx/shortcuts.md

```diff
@@ -0,0 +1,20 @@
+# shortcuts
+
+> Use the Terminal app to run shortcuts from command-line instead from the `Shortcuts` app.
```

Comment:
I think the terminal is implicit in the `command-line`:
```suggestion
> Run shortcuts from command-line instead from the `Shortcuts` app.
```

Reply:
I'd prefer something like this:

```suggestion
> Manage shortcuts from the command-line instead of the `Shortcuts` app.
```